docs/store.js

```diff
@@ -155,6 +155,8 @@ var firebaseCalendarStorage = function(params) {
         
         // Store the calendar into allCalendarIds, store calState
         var updates = {};
+        // Should I be updating the allCalendarIds every time I save? saving
```

Comment:
What's expensive is establishing the call to the database. Adding a little extra data probably doesn't matter much.